common/app/model/meta.scala

```diff
@@ -307,6 +310,7 @@ final case class MetaData(
     isNewRecipeDesign: Boolean = false,
     sensitive: Boolean = false,
     isFoundation: Boolean = false,
+    groupedNewslettersResponses: List[(String, List[NewsletterResponse])] = Nil,
```

Comment:
How come we need to add this to meta?

Reply:
It was the only way I could see to pass it through to the `Page` data in terms of how we actually render the page. I think that once we move tis page on to DCR we can tidy a lot of this  up

Reply:
Maybe the trick is not to use `SimplePage` but extend `StandalonePage` and add a new property for grouped newsletters?

Reply:
Good call! I'll look into that now